python/cudf/cudf/_lib/text.pyx

```diff
@@ -15,11 +16,24 @@ from cudf._lib.cpp.io.text cimport (
     byte_range_info,
     data_chunk_source,
     make_source,
+    make_source_from_bgzip_file,
     make_source_from_file,
     multibyte_split,
 )
 
 
+class BGZIPFile:
+    def __init__(self, filename, compression_offsets):
```

Comment:
Can we add a docstring here describing the parameters?

Reply:
I removed the class, so this became obsolete